bin/meson.build

```diff
@@ -0,0 +1,4 @@
+if get_option('mangoplot')
+    # runtime dependencies for the script: matplotlib and a GUI backed like PyQt5
+    install_data('mangoplot.py', install_dir: get_option('bindir'), rename: 'mangoplot', install_mode: 'rwxr-xr-x')
```

Comment:
The `install_data()` formatting is off - see the other `meson.build` for details.

The extra `option()` seems like an overkill - this is a simple python script which people can `rm` if they feel strongly against it.

Reply:
Should be good now